CHANGES.rst

```diff
@@ -21,6 +21,12 @@ ramp_fitting
 
 - Fixed memory leak in C-extension.[#281]
 
+saturation
+~~~~~~~~~~
+
+- Add option for using the readout pattern information to improve saturation flagging
+  in grouped data. [#283]
+
```

Comment:
```suggestion
```
#284 switched change log handling to `towncrier`, so this change log entry should be a file in `changes/`:
```shell
echo "[saturation] Add option for using the readout pattern information to improve saturation flagging in grouped data." > changes/283.bugfix.rst
```

(new PRs will have these instructions in the checklist)

Reply:
Done